utils/kata-manager.sh

```diff
@@ -815,7 +842,10 @@ handle_args()
 	shift $[$OPTIND-1]
 
 	[ -z "$kata_version" ] && kata_version="${1:-}" || true
-	[ -z "$containerd_version" ] && containerd_version="${2:-}" || true
+	[ -z "$containerd_flavour" ] && containerd_flavour="${2:-}" || true
+
+	[ -z "$containerd_flavour" ] && containerd_flavour="lts"
```

Comment:
Fine, or you could just set `local containerd_flavour='lts'` above to avoid the need for this line.